.gitlab-ci.yml

```diff
@@ -326,13 +326,18 @@ e2e:
     - mkdir -p $CI_PROJECT_DIR/test/e2e/scripts && cp $CI_PROJECT_DIR/install_script*.sh $CI_PROJECT_DIR/test/e2e/scripts
     # Retrieve common go deps
     - mkdir -p $GOPATH/pkg/mod && tar xJf modcache_e2e.tar.xz -C $GOPATH/pkg/mod && rm -f modcache_e2e.tar.xz
+    # Install orchestrion
+    - go install github.com/DataDog/orchestrion@v0.9.4
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
We could install `orchestrion` in our e2e runner image, defined in `test-infra-definitions. Can be a follow up and we can handle it